src/features/weather/weatherSlice.js

```diff
@@ -11,7 +11,8 @@ export const fetchWeatherForecast = createAsyncThunk(
       .fetchCurrentWeather({ query })
       .then((data) => weatherAPI.fetchWeatherForecast(data.data.coord))
       .then((forecastData) => {
-        forecastData.data.cityName = query;
+        forecastData.data.cityName =
+          query.charAt(0).toUpperCase() + query.slice(1);
```

Comment:
we don't have to mutate here, would be better if we returned new object like `return { ...forecastData, data: {...forecasData.data, cityName: query}}`